modules/bound-sa-tokens-configuring.adoc

```diff
@@ -122,8 +122,13 @@ spec:
 ----
 <1> A reference to an existing service account.
 <2> The path relative to the mount point of the file to project the token into.
-<3> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours.
+<3> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours. 
 <4> Optionally set the intended audience of the token. The recipient of a token should verify that the recipient identity matches the audience claim of the token, and should otherwise reject the token. The audience defaults to the identifier of the API server.
++
+[NOTE]
+====
+Currently, the `expirationSeconds` value is overridden by the `--service-account-extend-token-expiration` default of `true` up to one year to prevent unexpected failure during transition.
```

Comment:
@cbippley It is not clear to me what this means. Perhaps it is my ignorance. What does `service-account-extend-token-expiration= true` mean to the user?

Reply:
@mburke5678 It's a flag that defaults to `true` that overrides the `expirationSeconds` value that users can select for themselves. So if they want to know why that value isn't doing what they think it should, this is the explanation. Does that make sense?

Reply:
Do we want to expressly state that the `service-account-extend-token-expiration= true` parameter changes the service account token expiration to 1 year? That seems to be  huge change, from the default hour to a year. (Stating _up to one year_ is confusing, suggesting an open period of time.)  Also, what is the _transition_?

Reply:
Seems like it's one year, and not "up to one year" so I've changed it. And, as they haven't started the transition yet I removed the reference to it.